assets/src/js/docs/index.js

```diff
@@ -68,8 +74,73 @@ const docs = {
                 e.preventDefault();
             });
         });
-    }
+    },
+
+    // enables a rebrand toggle checkbox which switches between legacy and rebranded styling
+    _themeHandler: () => {
+        const toggleContainer = document.createElement('div'),
+            toggleBtn = document.createElement('btn');
+
+        toggleContainer.classList.add('sg-themeToggle');
+
+        toggleBtn.classList.add('o-btn', 'o-btn--outline', 'sg-themeToggle-btn');
+        toggleBtn.textContent = 'Switch to Menulog';
+        toggleBtn.addEventListener('click', docs._themeToggle);
+
+        toggleContainer.append(toggleBtn);
+        document.body.append(toggleContainer);
 
+        docs.themeBtn = $.first('.sg-themeToggle-btn');
+
+        const currentTheme = docs._getTheme();
+        if (currentTheme !== 'je') {
+            docs._setTheme(currentTheme);
+        }
+    },
+
+    _themeToggle: event => {
+        const isMenulog = event.target.innerText.toLowerCase().includes('menulog');
+        // if the stylesheet currently includes the Menulog prefix, change theme to JE
+        if (isMenulog) {
+            docs._setTheme('ml');
+        } else {
+            docs._setTheme('je');
+        }
+    },
+
+    _saveTheme: theme => {
+        if (window.localStorage && theme !== null) {
+            window.localStorage.setItem('docsTheme', theme);
+        }
+    },
+
+    _getTheme: () => {
+        const storedTheme = window.localStorage.getItem('docsTheme');
```

Comment:
Minor one but you probably want to check that localStorage exists before accessing anything on it. Similar to what you have done on line 112, it's also the first thing that gets called (before setTheme get's invoked) so it will error here first. Might be worth having a method to check that it exists then use that method to call the helper you need:

``` javascript
_getLocalStorageContext: (typeItem, name, value) => {
     if (window.localStorage) return window.localStorage[typeItem](name, value);
}

_getLocalStorageContext('getItem', 'foo');
_getLocalStorageContext('setItem', 'foo', 'baz');
```

Reply:
Good idea – I've updated to use this kind of method now